### PR TITLE
Starting client

### PR DESCRIPTION
**Pull request by JRJurman**
client work to show a canvas on the index page

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ app.get('/foo', (req, res) => res.send('bar'));
 app.get('/bar', (req, res) => res.send('foo'));
 
 app.use(express.static(path.join(__dirname, '/public')));
+app.use(express.static(path.join(__dirname, '/../dist')));
```
Comment (bbesmanoff) [inline server/app.js]: This will be changing.  Not sure exactly where yet

File: app/main.ts [hunk 2]
```diff
@@ -1,4 +1,22 @@
-import * as pixi from 'pixi.js';
+import * as PIXI from 'pixi.js';
 import RobotProxy from './robotproxy';
+window.onload = init;
```
Comment (bbesmanoff) [inline app/main.ts]: Why not just `window.onload = function() { ... }`?

File: app/main.ts [hunk 3]
```diff
@@ -1,4 +1,22 @@
...
+
+	var renderer = new PIXI.autoDetectRenderer(800, 600, {backgroundColor : 0x1099bb, view : document.getElementById("maincanvas")});
+	document.body.appendChild(renderer.view);
+
+	// create the root of the scene graph
+	var stage = new PIXI.Container();
+
+	// start animating
+	animate();
+	function animate() {
```
Comment (bbesmanoff) [inline app/main.ts:13]: Why is this it's own function?  Is it for a later-time?

File: server/app.js (hunk 1 above)
Comment (bbesmanoff) [inline server/app.js]: `path.join(__dirname, '..', 'app', 'dist')` when rebased against master

File: app/main.ts (hunk 3 above)
Comment (weswigham) [inline app/main.ts:13]: It gets passed as an argument to `requestAnimationFrame`.

Comment (bbesmanoff) [inline app/main.ts:13]: :+1: missed that

File: server/app.js [hunk 4]
```diff
@@ -10,6 +10,7 @@ app.get('/foo', (req, res) => res.send('bar'));
 app.get('/bar', (req, res) => res.send('foo'));
 
 app.use(express.static(path.join(__dirname, '/public')));
+app.use(express.static(path.join(__dirname, '/../app/dist')));
```
Comment (weswigham) [inline server/app.js]: Convert the string with path separators here and above to just be multiple arguments  to `path.join`.

**Comment by weswigham** [comment]
:shipit:

**Pull request merged**